src/Phinx/Db/Table/Column.php

```diff
@@ -151,7 +151,7 @@ public function getType()
     /**
      * Sets the column limit.
      *
-     * @param integer $limit
+     * @param mixed $limit integer|string - can be string for SET and ENUM
```

Comment:
What do you suggest here? New column option called 'values' for example?
That option could take both, string and array.